common/djangoapps/third_party_auth/tests/test_provider.py

```diff
@@ -128,6 +128,7 @@ def test_tpa_hint_provider_displayed_for_login(self):
         ]
         assert hidden_provider.provider_id in provider_ids
 
+    def test_tpa_hint_exp_hidden_provider_displayed_for_login(self):
```

Comment:
Breaking these tests apart due to weird caching issue.

Reply:
1. I think it means "explicitly". I'll update the names and docstrings.
2. I don't think its new, but if we don't break up the tests, it returns the first configured provider for some reason. I'll have to test this once again to find out the exact issue.